honeybee/radiance/datatype.py

```diff
@@ -51,36 +98,50 @@ def __set__(self, instance, value):
                                     ",".join(map(str, inputs)), value))
             setattr(instance, self._name, value)
 
-class RadDescrBoolFlags(RadDescrDefault):
+
+class RadianceBoolFlag(RadianceDefault):
     """Descriptor for all the inputs that are boolean flags or single
     alphabets
     """
     pass
 
 
-class RadDescrNum(RadDescrDefault):
+class RadianceNumber(RadianceDefault):
     """
     Descriptor to set numbers.
+    (Attributes inherited from base-class are explained there.)
```

Comment:
@mostaphaRoudsari Never thought of it that way. But totally makes sense. I will add them for all classes.